rotkehlchen/tests/unit/test_assets.py

```diff
@@ -244,7 +244,7 @@ def test_coingecko_identifiers_are_reachable(data_dir):
 def test_assets_json_meta():
     """Test that all_assets.json md5 matches and that if md5 changes since last
     time then version is also bumped"""
-    last_meta = {'md5': '271b43b7fcb2ae995260a869adcf8e48', 'version': 71}
+    last_meta = {'md5': 'bd4e22bc1bd65aef857bdbf9137b37d9', 'version': 71}
```

Comment:
Good that you did not upgrade the version here. Doing it would pull it in all other rotki instances. And right now (since v1.15.0) we are in the middle of chaging the way our supported assets are stored and updated so want to be safe.

See: #2550 and #2551

But I think that just adding a non-ethereum token into all_assets.json should be fine.

Reply:
Actually I was looking for a way to add custom currency in the UI, great to know this feature is planned